openssh/README.md

```diff
@@ -0,0 +1,84 @@
+# openssh
+
+## Maintainers
+
+The Habitat Maintainers: <humans@habitat.sh>
+
+## Usage
+
+## Testing
+
+### Build the openssh core plan
+
+From a terminal
+
+```bash
+hab pkg build .
+```
+
+### Test the openssh core plan
+
+Enter a habitat studio and perform the following steps (see [more information](#note)):
+
+```bash
+# set environment from the last openssh build
+source results/last_build.env
+
+# install inspec and dependency core/busybox
+hab pkg install -b core/busybox
+hab pkg install chef/inspec
+
+# install the openssh artifact
+hab pkg install -b results/$pkg_artifact
+
+# load the openssh habitat service
+hab svc load $pkg_ident
+```
+
+On successful load of the service, output will contain something like:
+
+```bash
+The core/openssh/7.5p1/20190327095344 service was successfully loaded
+```
+
+Remain in the same habitat studio for the next section
+
+### Run the tests
+
+```bash
+hab pkg exec chef/inspec inspec exec ./test
+```
+
+Sample output:
+
+```bash
+[7][default:/src:0]# hab pkg exec chef/inspec inspec exec ./test
+
+Profile: Habitat Core Plan openssh (openssh)
+Version: 0.1.0
+Target:  local://
+
+  ✔  ensure openssh binary service works: Bash command /hab/pkgs/core/openssh/7.5p1/20190326193307/sbin/sshd  --help 2>&1
+     ✔  Bash command /hab/pkgs/core/openssh/7.5p1/20190326193307/sbin/sshd  --help 2>&1 stdout should match /usage: sshd/
+     ✔  Bash command /hab/pkgs/core/openssh/7.5p1/20190326193307/sbin/sshd  --help 2>&1 stderr should eq ""
+  ✔  ensure openssh binary exists: File /hab/pkgs/core/openssh/7.5p1/20190326193307/sbin/sshd
+     ✔  File /hab/pkgs/core/openssh/7.5p1/20190326193307/sbin/sshd should exist
+  ✔  ensure sshd processes are all running and owned by correct owner: openssh processes
+     ✔  openssh processes /sbin/sshd process running
+     ✔  openssh processes /sbin/sshd should be owned by 'root'
+  ✔  ensure openssh service is listening: Port 2222
+     ✔  Port 2222 should be listening
+     ✔  Port 2222 protocols should include "tcp"
+     ✔  Port 2222 processes should include "sshd"
+
+
+Profile Summary: 4 successful controls, 0 control failures, 0 controls skipped
+Test Summary: 8 successful, 0 failures, 0 skipped
+```
+
+## Appendix
+
+### Note
+
+- chef/inspec assumes that several underlying tools, like lsof, netstat, etc are available on the PATH in the linux system under test.  core/busybox, which includes these utilities, must therefore be installed and binlinked in habitat studio environment prior to testing
```

Comment:
Nitpick/personal preference - I would backtick program names like `lsof` and `netstat`